2019-Q4-PyPI/RFP.md

```diff
@@ -0,0 +1,197 @@
+
+
+## What is the Request for Proposals period?
+
+**A Request for Proposals (RFP)** is a process intended to allow us (the Python Software Foundation)to collect proposals from potential contractors and select contractor(s) best suited to fulfill the specified work.
+
+After the RFP period closes we will evaluate the received proposals based on the [evaluation criteria](#evaluation-criteria), seek clarification from proposers as necessary, and select one or more contractors to complete the work specified in the [scope section](#scope).
+
+**Note**: This Request For Proposal document may be updated to reflect things that we learn during the process. The canonical version and history is available [here](https://github.com/python/request-for/blob/master/2019-Q1-PyPI/RFP.md).
+
+## How do I submit a proposal?
+
+Proposals should be submitted as [Portable Document Format (PDF)](https://en.wikipedia.org/wiki/PDF) files via email to [ernest@python.org](mailto:ernest@python.org).
+
+Proposals must be submitted before the end of the day **October 16, 2019 [AoE](https://www.timeanddate.com/time/zones/aoe)** (2019-10-16T12:00:00Z).
+
+### Elements of proposal
+
+A submission must, at a minimum, include the following elements:
+
+- Description of the team that will perform the work.
+  - General overview and names of individuals.
+  - Experience with [relevant technologies](#specific-technologies-used).
+  - Freelance or firm? Incorporation? Subcontracting?
+  - Free/Open Source Software experience?
+
+- Agreement to [project management and reporting](#project-management-and-reporting) requirements.
+
+- What Milestone(s) are you proposing for?
+  - We recommend proposing for all of the work in scope of Milestone 1, Milestone 2, or both.
+
+- Examples of similarly-complex projects completed previously.
+  - Referencing contributions to Free/Open Source projects is encouraged.
+
+- Project timeline estimates by milestone and task. These timelines should fit within our [project timeline](#project-timeline), however proposers may suggest adjustments based on their availability.
+  - Milestone 1 - Verifiable cryptographic signing of artifacts
+  - Milestone 2 - Systems for Automated Detection of Malicious Uploads
+
+- Project budget by milestone and task. Deviations from our [estimated budgets and caps](#estimated-budgets-and-caps) should be described and supported.
+  - Milestone 1 - Verifiable cryptographic signing of artifacts
+  - Milestone 2 - Systems for Automated Detection of Malicious Uploads
+
+## Evaluation criteria
+
+### Potential pass/fail:
+
+ - Contains all elements specified in the [elements of the proposal](#elements-of-proposal).
+ - Proposal is detailed enough to properly assess further criteria.
+ - Formatting and Submission requirements:
+   - Portable Document Format (PDF)
+   - Emailed to [ernest@python.org](mailto:ernest@python.org) by **October 16, 2019 [AoE](https://www.timeanddate.com/time/zones/aoe)** (2019-10-16T12:00:00Z)
```

Comment:
Not your pyfound address? (OK if so! just checking.)

Reply:
That's actually a pretty good idea to better ensure deliverability. Thanks for the suggestion.